cachito/workers/pkg_managers/gomod.py

```diff
@@ -148,6 +148,8 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
     :param list dep_replacements: dependency replacements with the keys "name" and "version"; this
         results in a series of `go mod edit -replace` commands
     :param dict git_dir_path: the full path to the application's git repository
+    :param str subpath: path to the module, relative to the root repository folder; "None" in case
+        the root module is being resolved
```

Comment:
I'm not sure... I tried to avoid having this parameter as well, but I did not figure out how. I'll check if we can remove it.

Reply:
It's probably needed for all the other methods, but for resolve_gomod specifically, I think you could just get the relative path between the two paths

Reply:
Well, it turns out that app_source_path is a `str`. But I went with this change anyway, since it allows us to not need to check if subpath == "." (which is what Cachito does when we're processing the root module).